From bac9da015020a4ef269401e353c921e38b77349d Mon Sep 17 00:00:00 2001
From: demmm <anke62@gmail.com>
Date: Sat, 30 Mar 2024 19:27:39 +0100
Subject: [PATCH] add bcachefs to filesystem options use in test cycle only,
 not ready for stable release, probably fstab chenges needed too

---
 src/libcalamares/partition/FileSystem.cpp      | 1 +
 src/modules/partition/core/PartitionLayout.cpp | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/libcalamares/partition/FileSystem.cpp b/src/libcalamares/partition/FileSystem.cpp
index c2aedc079f..b2b94c583d 100644
--- a/src/libcalamares/partition/FileSystem.cpp
+++ b/src/libcalamares/partition/FileSystem.cpp
@@ -41,6 +41,7 @@ prettyNameForFileSystemType( FileSystem::Type t )
     case FileSystem::Fat32:
     case FileSystem::Ntfs:
     case FileSystem::Xfs:
+    case FileSystem::Bcachefs:
     case FileSystem::Jfs:
     case FileSystem::Hfs:
     case FileSystem::Ufs:
diff --git a/src/modules/partition/core/PartitionLayout.cpp b/src/modules/partition/core/PartitionLayout.cpp
index b476a3181a..11e49c5a8d 100644
--- a/src/modules/partition/core/PartitionLayout.cpp
+++ b/src/modules/partition/core/PartitionLayout.cpp
@@ -178,6 +178,7 @@ PartitionLayout::setDefaultFsType( FileSystem::Type defaultFsType )
     case FileSystem::F2fs:
         // ok
         break;
+    case FileSystem::Bcachefs:
     case FileSystem::Fat16:
     case FileSystem::Hfs:
     case FileSystem::HfsPlus:
